apollo-compiler/src/test/graphql/com/example/measurements

```diff
@@ -240,6 +240,7 @@ kotlin-responseBased-subscriptions
 kotlin-responseBased-enums_as_sealed                                                              340
 kotlin-responseBased-operation_id_generator                                                       325
 kotlin-responseBased-merged_include                                                               323
+kotlin-responseBased-big_query                                                                    317
 kotlin-responseBased-enum_field                                                                   316
```

Comment:
I'm not sure why this line was added?

Reply:
2 PRs were merged in parallel:

* https://github.com/apollographql/apollo-kotlin/pull/4133/files added a test but did not update measurements (not sure why)
* https://github.com/apollographql/apollo-kotlin/pull/4132 changed the format of measurements

So in the end 4132 was merged without the new test that you just added. So thank you :) 

FWIW, this file is informative only. It's used to keep track of the size of codegen. It's handy to have it commited because it allows to easily bisect if there is a regression but we might as well put it in .gitignore